Code/Tools/FBuild/Documentation/docs/functions/test.html

```diff
@@ -12,18 +12,64 @@
 
 <h1>Test</h1>
     <div id='test' class='newsitemheader'>
-      Test
+      Summary
     </div>
     <div class='newsitembody'>
       <p>
-Defines the execution of an executable as a test. The executable must be previously defined with the Executable function.</p>
-<div class='code'>Test( alias )      ; (optional) Alias
+Defines the execution of an executable as a test.</p>
+<div class='code'>Test( alias )      // (optional) Alias
 {
-  .TestExecutable  ; Executable to execute
-  .TestOutput      ; Output file for captured test output
-  .TestArguments   ; (optional) Arguments to pass to test executable
-  .TestWorkingDir  ; (optional) Working dir for test execution
-  .TestTimeOut     ; (optional) TimeOut (in seconds) for test (default: 0, no timeout)
+  // Options
+  .TestExecutable  // The executable file to run that will execute the tests
+  .TestOutput      // Output file for captured test output
+  .TestArguments   // (optional) Arguments to pass to test executable
+  .TestWorkingDir  // (optional) Working dir for test execution
+  .TestTimeOut     // (optional) TimeOut (in seconds) for test (default: 0, no timeout)
+}
+</div>
+    </div>
+
+    <div class='newsitemheader'>
+      Details
+    </div>
+    <div class='newsitembody'>
+      <p>The Test function is used to create a target that is used to execute tests. It may depend on other FASTBuild targets via the <b>.TestExecutable</b> option.</p>
+      <p>When a test is run, its output (standard output and error channel) is captured and will be written to the file specified in <b>.TestOutput</b>. The captured output will not be shown directly (printed to FASTBuild's output) unless the test process has a non-zero return value. This is done in order to keep FASTBuild's output clean and reduce noise.</p>
+    </div>
+
+    <div class='newsitemheader'>
+      Options
+    </div>
+    <div class='newsitembody'>
+      <p><b>.TestExecutable</b> - String - (Required)</p>
+      <p>The executable file to run that will execute the tests.</p>
+      <p>Can either be a file path or the name of a target specified with the <a href='executable.html'>Executable</a> function.</p>
+      <hr>
+      <p><b>.TestOutput</b> - String - (Required)</p>
+      <p>Output file for captured test output.</p>
+      <p>When executing tests, FASTBuild will capture standard output channels of the executable and write them to this file when done.</p>
+      <hr>
+      <p><b>.TestArguments</b> - String - (Optional)</p>
+      <p>Arguments to pass to test executable.</p>
+      <hr>
+      <p><b>.TestWorkingDir</b> - String - (Optional)</p>
+      <p>Working dir for test execution.</p>
+      <hr>
+      <p><b>.TestTimeOut</b> - Integer - (Optional)</p>
+      <p>The amount of milliseconds to wait for a test to finish execution.</p>
```

Comment:
I've actually looked at the implementation of v.90 and saw that it was milliseconds, so I wrote that down in the documentation. I'm glad you're going for seconds instead because I think it makes more sense from a user's perspective! :+1: